rhel8/profiles/stig.profile

```diff
@@ -44,3 +44,6 @@ selections:
     - package_rsyslog-gnutls_installed
     - rsyslog_remote_tls
     - rsyslog_remote_tls_cacert
+    - sshd_rekey_limit
```

Comment:
Ummm... if the `stig.profile` extends the `ospp.profile` do we need to add the rule here?

Reply:
Well, I believe you can help me to answer this question :-)
 If we leave it in the current state, then in the STIG profile the rekey limit will stay at 512M 1H. If we leave this out, it will inherit the 1G 1H value. What is appropriate for rhel8 STIG?

Reply:
@redhatrises may be referring to the extra rule selection added: `- sshd_rekey_limit`

Reply:
Exactly, I don't think that we need to add `sshd_rekey_limit` or `var_rekey_limit_size` to the stig profile at all.